scripts/fast-reboot

```diff
@@ -407,10 +424,13 @@ if [[ "$REBOOT_TYPE" = "warm-reboot" || "$REBOOT_TYPE" = "fastfast-reboot" ]]; t
     fi
 fi
 
-# We are fully committed to reboot from this point on becasue critical
+# We are fully committed to reboot from this point on because critical
 # service will go down and we cannot recover from it.
 set +e
 
+# disable trap-handlers which were set before
+trap ' ' EXIT HUP INT QUIT TERM KILL ABRT ALRM
```

Comment:
I believe you should just use an empty string (`trap '' EXIT HUP INT QUIT TERM KILL ABRT ALRM`) or a hyphen (`trap - EXIT HUP INT QUIT TERM KILL ABRT ALRM`), depending on your intention (ignore vs reset traps).

Reply:
Change added now - using an empty string.